core/src/main/java/com/netease/arctic/utils/SnapshotFileUtil.java

```diff
@@ -126,4 +130,55 @@ private static void getDeleteFiles(
       LOG.error("close manifest file error", exception);
     }
   }
+
+  public static void getNativeIcebergDeleteFiles(
```

Comment:
How about rename method name to `getIcebergDeleteFiles` and make `getDeleteFiles` depende on this one?

Reply:
There is some difference between arctic and iceberg delete file. Arctic eq-delete file is datafile and all delete file is pos-delete file, but in iceberg there are both eq-delete and pos-delete type exist in table.

Reply:
Yes, but Getting all changed delete files from iceberg's entries table is also the same for Arctic and Iceberg Table as I  know, and they share many deplicate code now.